src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -48,14 +47,15 @@ fluid.default("fluid.parent", {
 
 ##### In 2.0 #####
 
-In Infusion 2.0 where relay components are introduced, sharing models no longer requires the change applier to be shared:
+In Infusion 2.0 where relay components are introduced, the change applier must not be configured separately - model sharing
```

Comment:
I think "must not be" - the user is pretty likely to corrupt this or future versions of the framework by attempting to directly configure any part of the model relay machinery. We do still support "changeApplierOptions" as a means of routing options to the applier - although currently we don't support any particularly useful options there.